### ISSUE TITLE
npm support

### ISSUE THREAD

**Issue by ducin** [lbovet/docson#38]
How about adding `docson` to npm registry, so that non-bower developers may use docson?

Many people say that `npm` will replace `bower` soon (and bower will become useless), one may agree with it or not. Anyway, some users would appreciate publishing `docson` in npm.

**Comment by rom1504** [comment]
I published a docson package there https://www.npmjs.com/package/docson

I removed some stuff so it make more sense as a package, so I'm not sure it makes sense to make a PR of that here.

I also added @lbovet to the package, if he wants to do anything about npm support for docson.

here is the branch where the changes are https://github.com/rom1504/docson/tree/npm and the important commit https://github.com/rom1504/docson/commit/c3f47e734f17d241123f42ac2c149f7831907eb0

In that form, it's fully usable with browserify.

**Comment by infiniteluke** [comment]
@lbovet It seems to me a v2.0.0 could be in order since adding NPM support is a breaking change. 

However, there are many other minor PRs not getting any love. I don't see much activity on this project, is there a roadmap of the project future?

**Comment by yanick** [comment]
In the npm version, I think we need to make sure that docson works in 4 incarnations:

[ ] as a npm library (duh)
[ ] as a cli tool that spin off an express app (`docson ./my/schema.json`)
[ ] as a widget
[ ] as a Swagger add-on

**Comment by rom1504** [comment]
Yes I agree. I did it a long time ago but I believe I remember my branch
only does the first point, I removed a bunch of files which are actually
probably needed for the other points. So I think that branch is more in the
"ideas" area than in something that should be a direct PR.

**Comment by yanick** [comment]
[branch is inspirational, not to be taken as-is] Yeah, I see it as such too. Mostly considering there have been a few clashing code changes since then too. But it does provide a good gist on which to base the process. :+1:
